### PR TITLE
chore(i18n): improved the Simplified Chinese translation

### PR DESCRIPTION
**Pull request by ttimochan**
This Pull Request modifies some of the Simplified Chinese translation; and for visual effect, adds the necessary spaces where Chinese and English are mixed. 

I can't access https://crowdin.com for some reason, so there's no way for me to submit content to the specified page

### CODE REVIEW

File: client/src/__locales/zh-cn.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
     "load_balancing_desc": "一次查询一台服务器。AdGuard Home 将使用加权随机算法来选择服务器，以便更常使用最快的服务器。",
     "bootstrap_dns": "Bootstrap DNS 服务器",
     "bootstrap_dns_desc": "DNS 服务器的 IP 地址，用于解析指定为上游的 DoH/DoT 解析器的 IP 地址。不允许评论。",
```
Comment (cxw620) [inline client/src/__locales/zh-cn.json:10]: 应该是 "不允许注释" 吧?

Comment (ttimochan) [reply client/src/__locales/zh-cn.json:10]: 原翻译如此。OK，我会改进翻译。

File: client/src/__locales/zh-cn.json [hunk 2]
```diff
@@ -2,15 +2,15 @@
     "client_settings": "客户端设置",
     "example_upstream_reserved": "指定为<0>特定域名</0>的上游服务器；",
     "example_upstream_comment": "注释。",
-    "upstream_parallel": "使用并行请求以同时查询所有上游服务器来加快解析速度。",
-    "parallel_requests": "并行请求",
+    "upstream_parallel": "使用并行的查询策略。同时查询所有上游服务器，Adguard Home 将会使用最快地上游服务器的查询结果",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:5]:
```suggestion
    "upstream_parallel": "使用并行的查询策略。同时查询所有上游服务器，AdGuard Home 将会使用最快的上游服务器的查询结果",
```

File: client/src/__locales/zh-cn.json [hunk 3]
```diff
@@ -2,15 +2,15 @@
...
+    "parallel_requests": "并行查询",
     "load_balancing": "负载均衡",
-    "load_balancing_desc": "一次查询一台服务器。AdGuard Home 将使用加权随机算法来选择服务器，以便更常使用最快的服务器。",
+    "load_balancing_desc": "一次查询一台服务器。AdGuard Home 将使用加权随机算法来选择服务器，以便更常使用最快上游的服务器。",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:8]:
```suggestion
    "load_balancing_desc": "一次查询一台服务器。AdGuard Home 将使用加权随机算法来选择服务器，以便更常使用最快的上游服务器。",
```

File: client/src/__locales/zh-cn.json [hunk 4]
```diff
@@ -2,15 +2,15 @@
...
     "bootstrap_dns": "Bootstrap DNS 服务器",
-    "bootstrap_dns_desc": "DNS 服务器的 IP 地址，用于解析指定为上游的 DoH/DoT 解析器的 IP 地址。不允许评论。",
-    "fallback_dns_title": "后备 DNS 服务器",
-    "fallback_dns_desc": "当上游 DNS 服务器没有响应时使用的后备 DNS 服务器列表。语法与上面的主要上游字段相同。",
-    "fallback_dns_placeholder": "每行输入一个后备 DNS 服务器",
+    "bootstrap_dns_desc": "DNS 服务器的 IP 地址，用于解析指定为上游的 DoH/DoT 解析器的 IP 地址。不允许添加注释。",
+    "fallback_dns_title": "备用 DNS 服务器",
+    "fallback_dns_desc": "当上游 DNS 服务器没有响应时使用的备用 DNS 服务器列表。语法与上面的相同。",
+    "fallback_dns_placeholder": "每行输入一个备用 DNS 服务器",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:13]:
```suggestion
    "fallback_dns_title": "后备 DNS 服务器",
    "fallback_dns_desc": "当上游 DNS 服务器没有响应时使用的后备 DNS 服务器列表。语法与上面的主要上游字段相同。",
    "fallback_dns_placeholder": "每行输入一个后备 DNS 服务器",
```

File: client/src/__locales/zh-cn.json [hunk 5]
```diff
@@ -67,7 +67,7 @@
     "ip": "IP 地址",
     "dhcp_table_hostname": "主机名",
     "dhcp_table_expires": "到期",
-    "dhcp_warning": "如果你想要启用内置的 DHCP 服务器，请确保在当前网络中没有其它起作用的 DHCP 服务器。否则，此操作可能会破坏已连接设备的网络连接！",
+    "dhcp_warning": "如果你想要启用内置的 DHCP 服务器，请确保在当前网络中没有其它起作用的 DHCP 服务器。否则，此操作可能会影响现有的网络设备！",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:70]:
```suggestion
    "dhcp_warning": "如果你想要启用内置的 DHCP 服务器，请确保在当前网络中没有其它起作用的 DHCP 服务器。否则，此操作可能会中断已连接设备的网络连接！",
```

File: client/src/__locales/zh-cn.json [hunk 6]
```diff
@@ -154,15 +154,15 @@
     "dns_settings": "DNS 设置",
     "dns_blocklists": "DNS 黑名单",
     "dns_allowlists": "DNS 白名单",
-    "dns_blocklists_desc": "AdGuard Home将阻止匹配DNS拦截清单的域名",
-    "dns_allowlists_desc": "来自 DNS 白名单的域名将被允许，即使它们位于任意黑名单中也是如此。",
+    "dns_blocklists_desc": "AdGuard Home 将阻止匹配 DNS 拦截清单的域名",
+    "dns_allowlists_desc": "在该 DNS 白名单的域名不会被拦截，即便是在 DNS 黑名单之中。",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:158]:
```suggestion
    "dns_blocklists_desc": "AdGuard Home 将阻止匹配 DNS 黑名单中的域名",
    "dns_allowlists_desc": "在 DNS 白名单中的域名不会被拦截，即便它们出现在任意 DNS 黑名单中。",
```

File: client/src/__locales/zh-cn.json [hunk 7]
```diff
@@ -207,9 +207,9 @@
     "custom_filter_rules": "自定义过滤器规则",
     "custom_filter_rules_hint": "请确保每行只输入一条规则。你可以输入符合 adblock 语法或 Hosts 语法的规则。",
     "system_host_files": "系统主机文件",
-    "examples_title": "范例",
-    "example_meaning_filter_block": "阻止 example.org 域名及其所有子域名；",
-    "example_meaning_filter_whitelist": "解除 example.org 及其所有子域名的封锁；",
+    "examples_title": "示例",
+    "example_meaning_filter_block": "拦截 example.org 域名及其所有子域名；",
+    "example_meaning_filter_whitelist": "解除 example.org 及其所有子域名的拦截；",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:212]:
```suggestion
    "example_meaning_filter_whitelist": "解除对 example.org 及其所有子域名的拦截；",
```

File: client/src/__locales/zh-cn.json [hunk 8]
```diff
@@ -220,7 +220,7 @@
     "example_upstream_udp": "常规 DNS（通过 UDP、主机名）；",
     "example_upstream_dot": "加密 <0>DNS-over-TLS</0>；",
     "example_upstream_doh": "加密 <0>DNS-over-HTTPS</0>；",
-    "example_upstream_doh3": "带有强制 <0>HTTP/3</0> 的加密 DNS-over-HTTPS，并且没有回退到 HTTP/2 或更低版本;",
+    "example_upstream_doh3": "带有强制 <0>HTTP/3</0> 的加密 DNS-over-HTTPS，不会降级到 HTTP/2 或更低版本;",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:223]:
```suggestion
    "example_upstream_doh3": "带有强制 <0>HTTP/3</0> 的加密 DNS-over-HTTPS，不会回退到 HTTP/2 或更低版本;",
```

File: client/src/__locales/zh-cn.json [hunk 9]
```diff
@@ -260,20 +260,20 @@
     "rule_added_to_custom_filtering_toast": "规则已添加到自定义过滤规则列表中 {{rule}}",
     "query_log_response_status": "状态： {{value}}",
     "query_log_filtered": "被 {{filter}} 过滤",
-    "query_log_confirm_clear": "你确定想要清除全部查询日志吗？",
-    "query_log_cleared": "查询日志已成功清除",
+    "query_log_confirm_clear": "你确定想要清空全部查询日志吗？",
+    "query_log_cleared": "查询日志已成功清空",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:264]:
```suggestion
    "query_log_confirm_clear": "你确定想要清除全部查询日志吗？",
    "query_log_cleared": "查询日志已全部清除",
```

File: client/src/__locales/zh-cn.json [hunk 10]
```diff
@@ -260,20 +260,20 @@
...
     "query_log_updated": "已成功更新查询日志",
-    "query_log_clear": "清除查询日志",
+    "query_log_clear": "清空查询日志",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:266]:
```suggestion
    "query_log_clear": "清除查询日志",
```

File: client/src/__locales/zh-cn.json [hunk 11]
```diff
@@ -331,7 +331,7 @@
     "install_settings_dns_desc": "您将需要使用以下地址来设置您的设备或路由器的 DNS 服务器：",
     "install_settings_all_interfaces": "所有接口",
     "install_auth_title": "身份认证",
-    "install_auth_desc": "您需要对 AdGuard Home 的网页管理界面配置密码认证。即使该 AdGuard Home 只能通过您的本地网络访问，避免 AdGuard Home 被不受限制地访问依旧十分重要。",
+    "install_auth_desc": "您需要对 AdGuard Home 的网页管理界面配置密码认证。即使该 AdGuard Home 只能通过您的本地网络访问，也应该使用密码认证来避免 Adguard Home 被他人意外修改。",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:334]:
```suggestion
    "install_auth_desc": "您需要对 AdGuard Home 的网页管理界面配置密码认证。即使该 AdGuard Home 只能通过您的本地网络访问，为它添加访问限制依旧十分重要。",
```

File: client/src/__locales/zh-cn.json [hunk 12]
```diff
@@ -459,12 +459,12 @@
     "access_disallowed_title": "不允许的客户端",
     "access_disallowed_desc": "CIDR、IP 地址或<a>客户端 ID</a> 的列表。如果已配置，则 AdGuard Home 将丢弃来自这些客户端的请求。如果允许的客户端已配置，此字段将会被忽略。",
     "access_blocked_title": "不允许的域名",
-    "access_blocked_desc": "不要将此功能与过滤器混淆。AdGuard Home 将排除匹配这些网域的 DNS 查询，并且这些查询将不会在查询日志中显示。在此可以明确指定域名、通配符（wildcard）和网址过滤的规则，例如 \"example.org\"、\"*.example.org\" 或 \"||example.org^\"。",
+    "access_blocked_desc": "不要将此功能与过滤器混淆。AdGuard Home 将排除匹配这些域名的 DNS 查询，并且这些查询将不会在查询日志中显示。在此可以明确指定域名、通配符（wildcard）和网址过滤的规则，例如 \"example.org\"、\"*.example.org\" 或 \"||example.org^\"。",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:462]:
```suggestion
    "access_blocked_desc": "不要将此功能与过滤器混淆。AdGuard Home 将排除匹配这些域的 DNS 查询，并且这些查询将不会在查询日志中显示。在此可以明确指定域名、通配符（wildcard）和网址过滤的规则，例如 \"example.org\"、\"*.example.org\" 或 \"||example.org^\"。",
```

File: client/src/__locales/zh-cn.json [hunk 13]
```diff
@@ -530,13 +530,13 @@
     "statistics_configuration": "统计配置",
     "statistics_retention": "统计保留",
     "statistics_retention_desc": "如果您减少该间隔的数值， 某些数据可能会丢失",
-    "statistics_clear": " 清除统计数据",
-    "statistics_clear_confirm": "您确定要清除统计数据？",
+    "statistics_clear": " 清空统计数据",
+    "statistics_clear_confirm": "您确定要清空统计数据？",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:534]:
```suggestion
    "statistics_clear": " 清除统计数据",
    "statistics_clear_confirm": "您确定要清除全部统计数据？",
```

File: client/src/__locales/zh-cn.json [hunk 14]
```diff
@@ -530,13 +530,13 @@
...
     "statistics_retention_confirm": "您确定要更改统计记录保留时间吗？ 如果您减少间隔时间的值， 某些数据可能会丢失。",
-    "statistics_cleared": "统计数据已成功清除",
+    "statistics_cleared": "统计数据已成功清空",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:536]:
```suggestion
    "statistics_cleared": "统计数据已全部清除",
```

File: client/src/__locales/zh-cn.json [hunk 15]
```diff
@@ -530,13 +530,13 @@
...
     "statistics_enable": "启用统计数据",
-    "ignore_domains": "忽略的网域（以换行符分隔）",
-    "ignore_domains_title": "被忽略的网域",
+    "ignore_domains": "忽略的域名（以换行符分隔）",
+    "ignore_domains_title": "被忽略的域名",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:539]:
```suggestion
    "ignore_domains_title": "被忽略的域",
```

File: client/src/__locales/zh-cn.json [hunk 16]
```diff
@@ -559,26 +559,26 @@
...
     "rewrite_AAAA": "<0>AAAA</0>：特殊值，保持来自上游的<0>AAAA</0>记录",
     "disable_ipv6": "禁用 IPv6 地址的解析",
-    "disable_ipv6_desc": "删除对 IPv6 地址（类型 AAAA）的所有 DNS 查询，并从 HTTPS 响应中删除 IPv6 提示。",
+    "disable_ipv6_desc": "丢弃对 IPv6 地址（类型 AAAA）的所有 DNS 查询，并从 HTTPS 响应中删除 IPv6 相关的信息。",
     "fastest_addr": "最快的 IP 地址",
-    "fastest_addr_desc": "查询所有 DNS 服务器并返回所有响应中速度最快的 IP 地址。因 AdGuard Home 必须等待全部 DNS 服务器均有所回应，因而会降低 DNS 查询的速度，但同时，此举将会改善总体的连接。",
-    "autofix_warning_text": "若您单击「修复」，AdGuardHome 将会配置您的系统以使用 AdGuardHome 的 DNS 服务器。",
-    "autofix_warning_list": "其将会进行如下工作：<0>停用系统DNSStubListener</0><0>设置DNS服务器地址为127.0.0.1</0><0>将/etc/resolv.conf的符号链接目标替换为/run/systemd/resolv/resolv.conf</0><0>停止DNSStubListener（重新加载系统解析服务）</0>",
-    "autofix_warning_result": "因此，默认情况下所有来自系统的DNS请求都将由AdGuardHome处理。",
+    "fastest_addr_desc": "查询所有 DNS 服务器并返回所有响应中速度最快的 IP 地址。因 AdGuard Home 必须等待全部 DNS 服务器响应，因而会降低 DNS 查询的速度，与此同时，此举将可能改善连接速度。",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:576]:
```suggestion
    "fastest_addr_desc": "查询所有 DNS 服务器并返回所有响应中速度最快的 IP 地址。由于 AdGuard Home 必须等待全部 DNS 服务器响应，这会降低 DNS 查询的速度，但此举将会在总体上改善连接速度。",
```

File: client/src/__locales/zh-cn.json [hunk 17]
```diff
@@ -559,26 +559,26 @@
...
+    "autofix_warning_text": "若您单击「修复」，AdGuard Home 将会配置您的系统以使用 AdGuard Home 的 DNS 服务器。",
+    "autofix_warning_list": "其将会进行如下工作：<0>停用系统 DNSStubListener</0><0>设置DNS服务器地址为 127.0.0.1</0><0>将/etc/resolv.conf的符号链接目标替换为/run/systemd/resolv/resolv.conf</0><0>停止 DNSStubListener（重新加载系统解析服务）</0>",
+    "autofix_warning_result": "因此，默认情况下所有来自系统的 DNS 请求都将由 AdGuard Home 处理。",
     "tags_title": "标签",
-    "tags_desc": "您可以选择与客户端对应的标记。标签可以包含在过滤规则中，并允许您更准确地应用它们。<0>了解更多</0>。",
+    "tags_desc": "您可以选择与客户端对应的标记。标签可以包含在过滤规则中，并允许您更准确地使用它们。<0>了解更多</0>。",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:581]:
```suggestion
    "tags_desc": "您可以选择与客户端对应的标记。标签可以包含在过滤规则中，并允许您更准确地应用它们。<0>了解更多</0>。",
```

File: client/src/__locales/zh-cn.json [hunk 18]
```diff
@@ -233,7 +233,7 @@
     "dns_test_warning_toast": "上游 “{{key}}” 不响应测试请求，可能无法正常工作",
     "unblock": "放行",
     "block": "拦截",
-    "disallow_this_client": "不允许这个客户端",
+    "disallow_this_client": "封禁这个客户端",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:236]:
```suggestion
    "disallow_this_client": "不允许这个客户端",
```

File: client/src/__locales/zh-cn.json [hunk 19]
```diff
@@ -594,22 +594,22 @@
     "check_service": "服务名称：{{service}}",
     "service_name": "服务名称",
     "check_not_found": "未在您的筛选列表中找到",
-    "client_confirm_block": "您确定要阻止客户端 \"{{ip}}\"?",
-    "client_confirm_unblock": "您确定要解除对客户端 \"{{ip}}\" 的封锁吗？",
+    "client_confirm_block": "您确定要封禁客户端 \"{{ip}}\"?",
+    "client_confirm_unblock": "您确定要解除对客户端 \"{{ip}}\" 的封禁吗？",
     "client_blocked": "客户端 \"{{ip}}\" 被成功拦截",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:599]:
```suggestion
    "client_blocked": "客户端 \"{{ip}}\" 被成功封禁",
```

File: client/src/__locales/zh-cn.json [hunk 20]
```diff
@@ -594,22 +594,22 @@
...
     "client_unblocked": "成功解锁客户端 \"{{ip}}\"",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:600]:
```suggestion
    "client_unblocked": "成功解除对客户端 \"{{ip}}\" 的封禁",
```

File: client/src/__locales/zh-cn.json [hunk 21]
```diff
@@ -624,17 +624,17 @@
     "blocklist": "黑名单",
     "milliseconds_abbreviation": "毫秒",
     "cache_size": "缓存大小",
-    "cache_size_desc": "DNS 缓存大小（单位：字节）。要关闭缓存，请留空。",
-    "cache_ttl_min_override": "覆盖最小TTL值",
-    "cache_ttl_max_override": "覆盖最大TTL值",
+    "cache_size_desc": "DNS 缓存大小（单位：字节）。如果要关闭缓存，请留空。",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:627]:
```suggestion
    "cache_size_desc": "DNS 缓存大小（单位：字节）。若要关闭缓存，请留空。",
```

File: client/src/__locales/zh-cn.json [hunk 22]
```diff
@@ -624,17 +624,17 @@
...
+    "cache_ttl_max_override": "覆盖最大 TTL 值",
     "enter_cache_size": "输入缓存大小（字节）",
     "enter_cache_ttl_min_override": "输入最小 TTL 值（秒）",
     "enter_cache_ttl_max_override": "输入最大 TTL 值（秒）",
     "cache_ttl_min_override_desc": "缓存 DNS 响应时，延长从上游服务器接收到的 TTL 值 (秒)。",
     "cache_ttl_max_override_desc": "设定 DNS 缓存条目的最大 TTL 值（秒）。",
     "ttl_cache_validation": "最小缓存 TTL 值必须小于或等于最大值",
     "cache_optimistic": "乐观缓存",
-    "cache_optimistic_desc": "即使条目已过期，也让 AdGuard Home 从缓存中响应，并尝试刷新它们。",
+    "cache_optimistic_desc": "即使缓存中部分域名的结果已过期，也让 AdGuard Home 从缓存中响应，并尝试刷新它们。",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:637]:
```suggestion
    "cache_optimistic_desc": "即使缓存中的结果已过期，也让 AdGuard Home 从缓存中响应，并尝试刷新它们。",
```

File: client/src/__locales/zh-cn.json [hunk 23]
```diff
@@ -656,9 +656,9 @@
     "served_from_cache": "{{value}}<i>(由缓存提供)</i>",
     "form_error_password_length": "密码必须至少有 {{value}} 个字符",
     "anonymizer_notification": "<0>注意：</0> IP 匿名化已启用。您可以在<1>常规设置</1>中禁用它。",
-    "confirm_dns_cache_clear": "您确定要清除 DNS 缓存吗？",
-    "cache_cleared": "已成功清除 DNS 缓存",
-    "clear_cache": "清除缓存",
+    "confirm_dns_cache_clear": "您确定要清空 DNS 缓存吗？",
+    "cache_cleared": "已成功清空 DNS 缓存",
+    "clear_cache": "清空缓存",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:661]:
```suggestion
    "confirm_dns_cache_clear": "您确定要清除 DNS 缓存吗？",
    "cache_cleared": "已成功清除全部 DNS 缓存",
    "clear_cache": "清除缓存",
```

File: client/src/__locales/zh-cn.json [hunk 24]
```diff
@@ -684,10 +684,10 @@
...
     "ignore_query_log": "在查询日志中忽略此客户端",
     "ignore_statistics": "在统计数据中忽略此客户端",
-    "schedule_services": "暂停服务拦截",
-    "schedule_services_desc": "配置服务拦截过滤器的暂停计划",
-    "schedule_services_desc_client": "为此客户端配置服务拦截过滤器的暂停计划",
-    "schedule_desc": "安排已拦截服务禁用的时间",
+    "schedule_services": "暂停拦截",
+    "schedule_services_desc": "暂停 Adguard Home 的拦截",
+    "schedule_services_desc_client": "一段时间内，对客户端在 DNS 黑名单的请求停止拦截",
+    "schedule_desc": "安排已拦截禁用的时间",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:690]:
```suggestion
    "schedule_services": "配置暂停拦截服务的计划",
    "schedule_services_desc": "配置何时暂停 AdGuard Home 的拦截服务",
    "schedule_services_desc_client": "在一定时间内，停止拦截该客户端在 DNS 黑名单中的请求",
    "schedule_desc": "应用暂停拦截服务的时间段",
```

**Review by kLiHz — CHANGES_REQUESTED** [review]

File: client/src/__locales/zh-cn.json (hunk 4 above)
Comment (ttimochan) [reply client/src/__locales/zh-cn.json:13]: 翻译成备用服务器更符合语境，不要过度机翻

**Comment by kLiHz** [comment]
I suggest we ensure the consistency between translations of a certain term. For example we should make sure the word "block" in all similar context are translated the same way. Similarly, please mind the upper/lower cases of words. In this case we should keep the word "AdGuard" the same as what it appears in other places.

By the way, my changes may not be consistent or precise either. They just represents my opinion.

**Comment by ttimochan** [comment]
> I suggest we ensure the consistency between translations of a certain term. For example we should make sure the word "block" in all similar context are translated the same way. Similarly, please mind the upper/lower cases of words. In this case we should keep the word "AdGuard" the same as what it appears in other places.


In the process of translation, I follow the principle of [信达雅](https://zh.wikipedia.org/zh-cn/%E4%BF%A1%E8%BE%BE%E9%9B%85), not machine translation, which is very important for users who use Simplified Chinese. If the user needs to see the original content, he can switch to English, but the translation into Simplified Chinese should be faithful and elegant!

File: client/src/__locales/zh-cn.json (hunk 4 above)
Comment (kLiHz) [reply client/src/__locales/zh-cn.json:13]: I don't think this is machine translating. I think in Chinese "后备资源" appears more often than "备用资源". The word "后备" has a sense of "backup" and "support", while "备用" slightly lacks it.

**Comment by fernvenue** [comment]
> I suggest we ensure the consistency between translations of a certain term. For example we should make sure the word "block" in all similar context are translated the same way.

Agree, that's a good example.

> Similarly, please mind the upper/lower cases of words. In this case we should keep the word "AdGuard" the same as what it appears in other places.

And yea, it should as same as English version all the time, use "AdGuard" not "Adguard".

**Comment by kLiHz** [comment]
@ttimochan But please note that some of your changes didn't even met the principle of faithful. 😋 I usually check my translation several times  before I submitted them, and it would be nice if we can all do so.

**Comment by ttimochan** [comment]
> @ttimochan But please note that some of your changes didn't even met the principle of faithful. 😋 I usually check my translation several times before I submitted them, and it would be nice if we all do so.

OK，I believe it should adhere to the Chinese context because certain terms may make perfect sense in an English context but could seem peculiar in a Chinese context.

**Comment by ttimochan** [comment]
> > I believe it should adhere to the Chinese context because certain terms may make perfect sense in an English context but could seem peculiar in a Chinese context.
> 


You are right, I will revise the translated content again.

File: client/src/__locales/zh-cn.json [hunk 25]
```diff
@@ -2,15 +2,15 @@
     "client_settings": "客户端设置",
     "example_upstream_reserved": "指定为<0>特定域名</0>的上游服务器；",
     "example_upstream_comment": "注释。",
-    "upstream_parallel": "使用并行请求以同时查询所有上游服务器来加快解析速度。",
-    "parallel_requests": "并行请求",
+    "upstream_parallel": "使用并行的查询策略。同时查询所有上游服务器，AdGuard Home 将会使用最快地上游服务器的查询结果",
```
[the remark this answers is not in the document]
Comment (ttimochan) [reply client/src/__locales/zh-cn.json:5]: Thanks.

File: client/src/__locales/zh-cn.json [hunk 26]
```diff
@@ -154,15 +154,15 @@
     "dns_settings": "DNS 设置",
     "dns_blocklists": "DNS 黑名单",
     "dns_allowlists": "DNS 白名单",
-    "dns_blocklists_desc": "AdGuard Home将阻止匹配DNS拦截清单的域名",
-    "dns_allowlists_desc": "来自 DNS 白名单的域名将被允许，即使它们位于任意黑名单中也是如此。",
+    "dns_blocklists_desc": "AdGuard Home 将阻止匹配 DNS 拦截清单的域名",
```
Comment (kLiHz) [inline client/src/__locales/zh-cn.json:157]:
```suggestion
    "dns_blocklists_desc": "AdGuard Home 将阻止匹配 DNS 黑清单的域名",
```

The "blocklist" is translated as "黑名单" above. I think it's better to keep the consistency.

Comment (ttimochan) [reply client/src/__locales/zh-cn.json:157]: OK.

**Comment by ainar-g** [comment]
Thanks for the suggestions!  Our team will inspect them and they should land in the next release.

In the future, please try to access and use CrowdIn.  That is the one place AdGuard uses for all its internationalization work, and we really would not like to split the effort.

**Pull request closed without merging**